src/main/java/io/jenkins/plugins/tuleap_api/client/Project.java

```diff
@@ -0,0 +1,5 @@
+package io.jenkins.plugins.tuleap_api.client;
+
+public interface Project {
```

Comment:
we do not need this interface, we only distribute UserGroup to the client of `tuleap-api`, the ProjectEntity does not need to hide behind an interface as this is the UserGroupEntity, hidden behind the UserGroup interface that makes sense as a whole.

Reply:
Can we delete this class ?